dev/containers/barberChat.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import io from 'socket.io-client';
+
+const socket = io('http://localhost:3000');
+
+// import { connect } from 'react-redux';
+
+// barber component joins room automatically when view is loaded
+// this way the user view can grab the corresponding barber email
+// when trying to chat and user view will emit a message to that room
+
+class BarberChat extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      term: '',
+      email: 'snypper@io.com',
+    };
+  }
+  componentDidMount() {
+    socket.emit('join', { email: 'snypper@io.com' });
```

Comment:
Can the second argument on .emit just be `this.state.email`? You'd probably have to save the BarberChat context in a variable instead of using `this` but I think you should know what I mean.